src/main/java/seedu/command/interpreter/AttendanceCommandInterpreter.java

```diff
@@ -78,14 +78,14 @@ public Command decideCommand(String commandDescription) throws DukeException {
             eventName = ui.getEventNameForAttendance();
             attendances = getAttendance(eventName);
             return new ClearAttendanceList(attendances, eventName);
-        case "sort":
+        case "sort/by/name":
             eventName = ui.getEventNameForAttendance();
             attendances = getAttendance(eventName);
-            return new SortAttendanceListByName(attendances);
-        case "sort/":
+            return new SortAttendanceListByName(attendances, eventName);
+        case "sort/by/status":
```

Comment:
I don't think sort/by/name and sort/by/status is consistent with our flag naming convention...

Perhaps use `sort by/name` and `sort by/status`? Just a suggestion to follow our flag naming convention

Reply:
I can change the sort naming, but do take note that I'm not using flags for this.

Reply:
Would it be better if the user input "attendance sort", then the next line determines what type of sort?

Reply:
Amended in https://github.com/AY1920S2-CS2113T-T12-4/tp/pull/113/commits/1c2f3042c1e51792475b91b7d5b9127e47db9a9a